pkg/healthcheck/healthcheck.go

```diff
@@ -601,8 +607,10 @@ func (hc *HealthChecker) Add(categoryID CategoryID, description string, hintURL
 			checkers: []checker{
 				checker{
 					description: description,
-					check:       check,
-					hintURL:     hintURL,
+					check: func(_ context.Context) error {
+						return check()
```

Comment:
i think for consistency in testing (which i think there are a few to fix ;) ), this should be:
```go
func (hc *HealthChecker) Add(categoryID CategoryID, description string, hintURL string, check func(context.Context) error) {
...
	check: check,
```